shared/templates/create_auditd_lineinfile.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python2
```

Comment:
Why `python2`?

Reply:
Honestly, it was copied from `create_services_enabled.py` so it came from there. It is probably due to some RHEL6 support, but I'm not sure. Looking at `create_*.py` files, some of them use `#!/usr/bin/env python2`, some of them use `#!/usr/bin/python2` and some of them don't have the shebang at all. It is probably safe to remove.

Reply:
I removed from the file `create_lineinfile.py` which is the super class file for both `auditd` and `sshd` templates. Marking as resolved.